com.beavernotes.beavernotes.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: com.beavernotes.beavernotes
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: beaver-notes
+finish-args:
+  - --device=dri
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=home
+  - --filesystem=/mnt
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
```

Comment:
Electron does not default to native wayland, it's better to use just `--socket=x11`. See the note and explanation below https://docs.flatpak.org/en/latest/electron.html#sandbox-permissions

Reply:
I actually think it's better to keep it as-is instead of using just `--socket=x11`, since with this finishing argument, Beaver doesn't start on Ubuntu 24.04, prompting the following error:
`[2:0416/100251.291084:ERROR:wayland_connection.cc(193)] Failed to connect to Wayland display: No such file or directory (2)
[2:0416/100251.291107:ERROR:ozone_platform_wayland.cc(259)] Failed to initialize Wayland platform
`

Reply:
Quite possibly electron/chromium changed something again since I wrote that guide.